neurox/interpretation/linear_probe.py

```diff
@@ -89,6 +89,9 @@ def _train_probe(
     is trained with Cross Entropy loss for classification tasks and a linear
     regression model is trained with MSE loss for regression tasks. The
     optimizer used is Adam with default ``torch.optim`` hyperparameters.
+    On GPU, the probe is trained with mixed precision (amp). 
```

Comment:
I think this comment needs to be updated with our new non-mixed precision training approach?